design/api-specification-design.md

```diff
@@ -11,40 +11,29 @@ The `report` request obtains a report of their account.
 
 #### Response
 - 200: a `report statement` will be sent:
-
-```json
-{
-    "data" : {
-        "report" : "---report statement-(see below)---"
-     }
-}
-```
-
-An example `report statement` for status code 200 :
 ```
 {
-"report": [
-  {
-    "date": ...,
-    "amount": ...,
-    "balance": ...,
-    "description": ...,
-  },
-  ...
-]
+"data" : {
```

Comment:
I still don't think we need this outer 'data' field.

Let's have an actual example here. You can document each field individually outside of this payload if you want to. Think of this as something you can copy and paste as a valid payload.

Also, let's be consistent with indentation (e.g. 2 spaces):

```json
{
  "report": [
    {
      "date": "2018-11-28T17:34:45Z",
      "amount": 1234,
      "balance": 12345678,
      "description": "blah"
    }
  ]
}
```

Applies below as well.